include/picongpu/initialization/ParserGridDistribution.hpp

```diff
@@ -77,26 +76,39 @@ class ParserGridDistribution
 
     /** Get local Size of this dimension
      *
-     *  \param[in] gpuPos as integer in the range [0, n-1] for this dimension
+     *  \param[in] devicePos as integer in the range [0, n-1] for this dimension
      *  \return uint32_t with local number of cells
      */
     uint32_t
-    getLocalSize( const int gpuPos ) const
+    getLocalSize( const int devicePos ) const
     {
         value_type::const_iterator iter = parsedInput.begin();
-        // go to last gpu of this block b{n}
+        // go to last device of this block b{n}
         int i = iter->second - 1;
 
-        while( i < gpuPos )
+        while( i < devicePos )
         {
             ++iter;
-            // go to last gpu of this block b{n}
+            // go to last device of this block b{n}
             i += iter->second;
         }
 
         return iter->first;
     }
 
+    /** Verify the number of blocks matches the devices
```

Comment:
`blocks` should be named  `domains` because we splitting the global domain into subdomains.

Reply:
agreed, just took the `block` naming already prevalent in this file :)